controllers/hcloudremediation_controller.go

```diff
@@ -55,7 +55,7 @@ type HCloudRemediationReconciler struct {
 func (r *HCloudRemediationReconciler) Reconcile(ctx context.Context, req reconcile.Request) (_ reconcile.Result, reterr error) {
 	log := ctrl.LoggerFrom(ctx)
 
-	// Fetch the Hetzner bare metal host instance.
+	// Fetch the Hetzner hcloud host instance.
```

Comment:
this is not a host at all.. Can you change it to hcloud remediation?

Reply:
@janiskemper I removed the comment. I think these two lines (create empty object, then use r.Get()) are very common and not hard to read. Is this ok for you?

Reply:
yes